File: _persist/temp/venda/CSharp/Aliqicms.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("ALIQICMS")]
    public class Aliqicms : CollectionItem
    {
        [Campo("UF_ORIGEM", CampoTipo.tfKey)]
        public string Uf_Origem { get; set; }
        [Campo("UF_DESTINO", CampoTipo.tfKey)]
        public string Uf_Destino { get; set; }
        [Campo("U_VERSION", CampoTipo.tfNul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.tfReq)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.tfReq)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("PR_ALIQUOTA", CampoTipo.tfReq)]
        public double Pr_Aliquota { get; set; }
    }
}

File: _persist/temp/venda/Java/Aliqicms.java
package MORM.Java.Models;

import java.util.Date;
import MORM.Java.Classes;

@Tabela("ALIQICMS")
public class Aliqicms extends CollectionItem
{
    @Campo("UF_ORIGEM", CampoTipo.tfKey)
    public String Uf_Origem;
    @Campo("UF_DESTINO", CampoTipo.tfKey)
    public String Uf_Destino;
    @Campo("U_VERSION", CampoTipo.tfNul)
    public String U_Version;
    @Campo("CD_OPERADOR", CampoTipo.tfReq)
    public Integer Cd_Operador;
    @Campo("DT_CADASTRO", CampoTipo.tfReq)
    public DateTime Dt_Cadastro;
    @Campo("PR_ALIQUOTA", CampoTipo.tfReq)
    public Float Pr_Aliquota;
}